lib/puppet/functions/nested_values.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# This function will return list of Hash values, the return value will be Array
+# NOTE : This function is expecting only Hash and return value will be Array
+#
+# @example :
+# $hash = {
+#   "key1" => "value1",
+#   "key2" => { "key2.1" => "value2.1"}
+# }
+# $hash.nested_values
+#
+# Output : ["value1", "value2.1"]
+#
+Puppet::Functions.create_function(:nested_values) do
```

Comment:
What happened to all new functions being namespaced??

Reply:
Suggest namespacing it _without_ creating a shim as this hasn't made it to a release yet.

Reply:
Damned!  Yes, to all of this :sleepy: